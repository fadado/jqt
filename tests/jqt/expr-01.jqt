The value of pi is {{.pi}}.
