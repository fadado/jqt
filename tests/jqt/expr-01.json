{ "pi": 3.14159265359 }
