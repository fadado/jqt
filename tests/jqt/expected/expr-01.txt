The value of pi is 3.14159265359.

